packages/itwin/property-grid/src/components/MultiElementPropertyGrid.scss

```diff
@@ -22,10 +22,18 @@
 }
 
 .property-grid-react-animated-tab {
-  transition: transform 500ms;
+  transition: transform 400ms ease-out;
   position: absolute;
   top: 0;
   left: 0;
   height: 100%;
   width: 100%;
 }
+
+.animate-right {
```

Comment:
Can you give animate-left and animate-right a more specific name like property-grid-animate-left/right, or nest it:
.property-grid-react-animated-tab {
  &.animate-right {}
  &.animate-left {}
}